### PR TITLE
Slim down issue template appearance

### PR DESCRIPTION
**Pull request by liggitt**
Moves the description to the top so it appears in email summaries
Change to a much smaller header to make actual text more readable

### CODE REVIEW

**Comment by deads2k** [comment]
> Moves the description to the top so it appears in email summaries
> Changes headers to bold

Is lack of a template a problem that we've had for the a significant percentage of issues?  I noticed issues based on this template this morning and thought that it significantly reduced the readability of the issue itself.  Certainly dev-to-dev, this is a waste of time/space.

**Comment by liggitt** [comment]
I don't know, but this helps appearance, imo

**Comment by liggitt** [comment]
@bparees thoughts?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+
+##### Steps To Reproduce
 
 1. [step 1]
 2. [step 2]
 
-## Current Result
+##### Current Result
+
+...
```
Comment (bparees) [inline .github/ISSUE_TEMPLATE.md]: if we're going to bikeshed, i'd rather not have "..." here since everyone will have to delete it.

(same below)

**Comment by bparees** [comment]
@deads2k is clearly in the @stevekuznetsov camp.

@deads2k you're free to delete the template and go free-form, but for the community coming to us with problems, this helps insure they actually provide the information we need to help them with their problem.

developers should be using bugzilla and trello anyway not issues, since issues are not well integrated into our tracking/QE processes.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (liggitt) [inline .github/ISSUE_TEMPLATE.md]: Sure

**Comment by liggitt** [comment]
limited changes to moving the description to the top and decreasing the header size

**Comment by liggitt** [comment]
[merge]

**Comment by mfojtik** [comment]
@bparees @deads2k i saw couple issues opened using this template and I don't think it makes the issue reading more complex... in fact it provides some clues for users where and how to get useful informations... for dev-dev I agree it is pain (ctrl-a+del ;-)

**Pull request merged**